parquet/src/file/metadata.rs

```diff
@@ -462,6 +463,16 @@ impl ColumnChunkMetaData {
         self.statistics.as_ref()
     }
 
+    /// Returns `true` if this column chunk contains a bloom filter offset, `false` otherwise.
+    pub fn has_bloom_filter_offset(&self) -> bool {
```

Comment:
Maybe 'has_bloom_filter' to align with 'has_index_page' and 'has_dictionary_page'